From a452138a1dd274bfad381a701729783360dc86fb Mon Sep 17  00:00:00 2001
From: Maciej Borzecki <maciej.borzecki@open-rnd.pl>
Date: Tue, 5 Jan 2016 22:04:17 +0100
Subject: [PATCH] fix cross compilation on i586 targets

Remove offending -Wl,-rpath-link that may cause host libraries to be picked
during linking. The patch applies a fix to configure.in. So as not to
regenerate configure, similar fix is applied there.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Maciej Borzecki <maciej.borzecki@open-rnd.pl>

Rebase to 52.8.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 js/src/old-configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/js/src/old-configure
+++ b/js/src/old-configure
@@ -3833,7 +3833,7 @@ AS='$(CC)'
 AS_DASH_C_FLAG='-c'
 MOZ_USER_DIR=".mozilla"
 
-MOZ_FIX_LINK_PATHS="-Wl,-rpath-link,${DIST}/bin -Wl,-rpath-link,${prefix}/lib"
+MOZ_FIX_LINK_PATHS="-Wl,-rpath-link,${DIST}/bin"
 
 
 
